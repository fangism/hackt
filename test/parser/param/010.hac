// "010.hac" should fail: type mismatch

pbool thirteen = 13;

