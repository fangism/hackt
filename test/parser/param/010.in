// should fail: type mismatch

pbool thirteen = 13;

