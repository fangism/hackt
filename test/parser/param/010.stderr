Cannot initialize a bool parameter with a pint-const expression, ERROR!  
Error initializing item 1 of alias-list.  
HALT: at least one error in alias list.  [3:16..18]
